src/lib/util/Cache/MemberTags.ts

```diff
@@ -48,16 +54,22 @@ export class MemberTags extends Collection<string, MemberTag> {
 	public async fetch(): Promise<this>;
 	public async fetch(id?: string): Promise<MemberTag | null | this> {
 		if (typeof id === 'undefined') {
-			const members = await this.guild.members.fetch();
-			for (const member of members.values()) this.create(member);
+			if (this.kFetchAllPromise === null) {
+				this.kFetchAllPromise = this.guild.members.fetch().then(members => {
+					for (const member of members.values()) this.create(member);
+					this.kFetchAllPromise = null;
+				});
+			}
+
+			await this.kFetchAllPromise;
 			return this;
 		}
 
 		const existing = this.get(id);
 		if (typeof existing !== 'undefined') return existing;
 
 		try {
-			const member = await this.guild.members.fetch(id);
+			const member = await this.kRequestHandler.push(id);
```

Comment:
Can't this block also be changed to use the new util function to ignore expected DAPI errors?

Reply:
Resolved in 05541f9.